File: Top 100 Codes/001-check if a Number Is Positive Or Negative/Method 1- Using Brute Force/PositiveAndNegativeNumber.cpp
#include<iostream>
using namespace std;

int main()
{
    int num;
    //take input
    cout<< "Enter a number:";
    cin>>num;

    //Conditions to check if the number is negative or positive
    if (num > 0)
        cout << "The number is positive";
    else if (num < 0)
        cout<< "The number is negative";
    else
        cout << "Zero";

    return 0;
}

File: Top 100 Codes/001-check if a Number Is Positive Or Negative/Method 1- Using Brute Force/Positive_And_Negative_Number.java
import java.util.Scanner;

public class Positive_And_Negative_Number {
    public static void main(String[] args) {
        //Brute force
         Scanner sc = new Scanner(System.in);
         System.out.println("Enter a Number: ");
         int num = sc .nextInt();

         //Conditiion to check if the number is positive or negative
         if (num > 0)
             System.out.println("The number is Positive");
        else if (num < 0)
             System.out.println("The number is Negative");
        else
             System.out.println("zero");

    sc.close();

        
    }
    
}